clarity/contracts/arkadiko-auction-engine-v1-1.clar

```diff
@@ -258,12 +258,16 @@
 ;; @desc calculate the discounted auction price on the (dollarcent) price of the collateral
 ;; @param price; the current on-chain price
 ;; @param auction-id; the ID of the auction in which the collateral will be sold
-(define-read-only (discounted-auction-price (price uint) (auction-id uint))
+(define-read-only (discounted-auction-price (price uint) (decimals uint) (auction-id uint))
   (let (
     (auction (get-auction-by-id auction-id))
     (discount (* price (get discount auction)))
+    (total-decimals (if (> decimals u0)
```

Comment:
Is this just to make sure or did you find an actual use case for this?

Reply:
If the oracle cannot find a price for the token passed, it returns 0 for price and 0 for decimals. Price can be 0 (then it just returns 0 I think). But for decimals, you would divide by 0 and it crashes